web/app/components/since.js

```diff
@@ -1,26 +1,29 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect } from "react";
 
-export default function Since( { props }) {
+export default function Since({ props }) {
   const [time, setTime] = useState(new Date());
-  const since = new Date(props);
+
 
   useEffect(() => {
     const timer = setInterval(() => {
       setTime(new Date());
-    }, 1000)
+    }, 1000);
     return () => {
-      clearInterval(timer)
-    }
-  }, [])
-  
+      clearInterval(timer);
+    };
+  }, []);
+
   // const daysSince = Math.floor((time - since) / (1000 * 24 * 60 * 60));
-  const secondsSince = Math.floor((time - since) / (1000));
-  
-  return ( 
+  const secondsSince = props
```

Comment:
reads weird that we're calling this `props`, maybe give it a more semantic name?

Reply:
Changing it to lastEaten